OoyalaSkinSampleApp/app/src/main/java/com/ooyala/sample/lists/MultiAudioListActivity.java

```diff
@@ -0,0 +1,95 @@
+package com.ooyala.sample.lists;
+
+import android.app.Activity;
+import android.content.Intent;
+import android.os.Bundle;
+import android.view.View;
+import android.widget.AdapterView;
+import android.widget.AdapterView.OnItemClickListener;
+import android.widget.ArrayAdapter;
+import android.widget.ListView;
+
+import com.ooyala.sample.R;
+import com.ooyala.sample.players.MultiAudioActivity;
+import com.ooyala.sample.utils.PlayerSelectionOption;
+
+import java.util.LinkedHashMap;
+import java.util.Map;
+
+public class MultiAudioListActivity extends Activity implements OnItemClickListener {
+
+  public final static String getName() {
+    return MultiAudioListActivity.class.getSimpleName();
+  }
+
+  private static Map<String, PlayerSelectionOption> selectionMap;
+  private ArrayAdapter<String> selectionAdapter;
+
+  /**
+   * Called when the activity is first created.
+   */
+  @Override
+  public void onCreate(Bundle savedInstanceState) {
+    super.onCreate(savedInstanceState);
+    setTitle(getName());
+
+    String pcode = "k0a2gyOt0QGNJLSuzKfdY4R-hw2b";
+    String domain = "http://www.ooyala.com";
+
+    selectionMap = new LinkedHashMap<>();
+
+    selectionMap.put("MT-1 ENG/GER", new PlayerSelectionOption("wxNjRwZTE6bxMBkrQbfM_BwokGf0pyOm", pcode, domain, MultiAudioActivity.class));
```

Comment:
For all the assets in this list, for the name/title of the asset add a more meaningful name for it, so that a new user can understand what the asset is supposed to have by just reading the title.